test/promql/tests/application_errors_test.yaml

```
@@ -0,0 +1,102 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.application_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 5m
+    input_series:
+
+      # HAS is experiencing application deletion failure rate of 100%, so it will be alerted
+      - series: 'has_application_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
+        values: '1 2 3'
+      - series: 'has_application_failed_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
+        values: '1 2 3'
```

I think it will make better sense to have values here collected for at least 65 minutes.
```suggestion
      - series: 'has_application_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
        values: '1+1x64'
      - series: 'has_application_failed_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
        values: '1+1x64'
```